infrastructure/lambda/template.yaml

```diff
@@ -103,6 +103,14 @@ Mappings:
       integration: 2
       production: 2
 
+  ThirdPartyApiMapping:
```

Comment:
@MMcC-GDS Could we make this DVADigitalEnabled that way we can do a combination of if DVADigitalEnabled and header present use DVADigital else use existing.

Just trying to ensure we can enable and test the new method without blocking the old on the environments

Reply:
Refactored and updated related fields and values.